example_workspace/src/nodelet_sample_app/jni/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        applicationId = 'com.example.hello_ros'
+        minSdkVersion 24
+        targetSdkVersion 24
+        externalNativeBuild {
+            cmake {
+                arguments   '-DANDROID_STL=c++_shared',
+                            '-DCMAKE_INSTALL_PREFIX=' + System.getenv('TARGET_DIR'),
+                            '-DCMAKE_FIND_ROOT_PATH=' + System.getenv('TARGET_DIR'),
+                            '-DCMAKE_PREFIX_PATH=' + System.getenv('TARGET_DIR'),
```

Comment:
Please see my other comments above.

Reply:
Yes, I have tried the other build option you have commented. I will update the examples with that